app/views/posts/index.html.erb

```diff
@@ -1 +1,22 @@
-<h1>Here is a list of posts for a given user</h1>
\ No newline at end of file
+<h1>Here is a list of posts all posts for the user</h1>
+
+<div class="users-container">
+  <img class="user-photo" src=""><br>
+  <h2><%= User.find(params[:id]).name %></h2><br>
+  <p class="text-usercard-index">Number of Posts: <%= User.find(params[:id]).posts_counter %></p><br><br>
+</div>
+ <% @posts.each_with_index do |post, index| %>
+  <div class="posts-container">
+    <div class="posts-title-container">
+      Post #<%= index + 1 %>:<br><br>
+      Title: <%= post.title %><br><br>
+      <%= post.text %><br>
+    </div>
```

Comment:
- [ ] Still in line with the previous review. There should be a link that directs a user to a post. Please add the link.